package-lock.json

```diff
@@ -0,0 +1,2665 @@
+{
+  "requires": true,
+  "lockfileVersion": 1,
```

Comment:
This file should not be in the repository.

Reply:
Hehehe... it keeps sneaking in each time whe get distracted. I'll remove it.